ramls/check-in-by-barcode-response.json

```diff
@@ -77,10 +77,18 @@
           "properties": {
             "name": {
               "description": "Name of the item status",
-              "type": "string"
+              "type": "string",
+              "$ref": "schema/item-status-names.json"
+            },
+            "date": {
+              "type": "string",
+              "description": "Date time when status was last changed",
+              "format": "date-time",
+              "readonly": true
             }
           },
-          "additionalProperties": false
+          "additionalProperties": false,
+          "required": ["name"]
```

Comment:
What does a required property mean in a schema used only for responses, that it is guaranteed to be there?

Reply:
Yes, I think so. Just for documentation purpose.

Reply:
This would mean that not including the name of the status was a breaking compatibility change?

What does the client gain from knowing this is required?

Reply:
My main thought was that since Item.status will always be present, then it always be included in the response.

Reply:
> My main thought was that since Item.status will always be present, then it always be included in the response.

I think that starts to couple the public `circulation` API with expectations from the `item-storage` API.

If this is only for documentation purposes, I think we should consider deferring this change until we better understand the impact on maintainability of designs like this.